csrankings-l.csv

```diff
@@ -794,6 +794,8 @@ Luís Caldas de Oliveira,Universidade de Lisboa,https://www.l2f.inesc-id.pt/wiki
 Luís Carlos Erpen De Bona,UFPR,https://web.inf.ufpr.br/bona,YJL1PhUAAAAJ
 Luís Carriço,Universidade de Lisboa,https://ciencias.ulisboa.pt/perfil/lmcarrico,MLg1ZJMAAAAJ
 Luis Ceze,University of Washington,https://www.cs.washington.edu/people/faculty/luisceze,KzESVKwAAAAJ
+Luis Cruz,TU Delft,https://luiscruz.github.io/,O13oaH0AAAAJ

```

Comment:
Looks like wrong DBLP: see https://dblp.org/pid/96/2486-2.html.

Reply:
Should I add alias "Luis Cruz 0002" and "Luís Cruz 0002"?

Reply:
You should only have entries corresponding to the above aliases.

Reply:
Please remove the "old" entries for Luis Cruz (the ones without the disambiguation suffixes).